### ISSUE TITLE
It wan an error when load fluent-plugin-norikara using fluentd v0.14.1

### ISSUE THREAD

**Issue by Konboi** [norikra/fluent-plugin-norikra#14]
this is error log

```
/opt/fluentd/ruby/2.3.0/lib/ruby/2.3.0/rubygems/specification.rb:2284:in `raise_if_conflicts': Unable to activate msgpack-rpc-over-http-0.0.7, because msgpack-0.7.6 conflicts with msgpack (~> 0.5.11) (Gem::ConflictError)
```

fluentd  v0.14.1 depends msgpack over v0.7.0 (https://github.com/fluent/fluentd/blob/v0.14.1/fluentd.gemspec#L22) 

but msgpack-rpc-over-http depends msgpack v0.5.11
(https://github.com/authorNari/msgpack-rpc-over-http/blob/v0.0.7/msgpack-rpc-over-http.gemspec#L19)

please check it.

**Comment by tagomoris** [comment]
Ah, it should be fixed.

**Comment by tagomoris** [comment]
https://github.com/authorNari/msgpack-rpc-over-http/pull/7

**Comment by tagomoris** [comment]
I released `v0.4.1` to solve this problem. It can be installed with Fluentd `v0.14.1` as far as I confirmed.
Please reopen this issue if you still have any problems.l

**Comment by tagomoris** [comment]
I found problems right after releasing `v0.4.1`, so I released `v0.4.2`. I should work well with Fluentd v0.14.x.
